flow_action_components/CollectionProcessors/force-app/main/default/classes/GetChildCollection.cls

```diff
@@ -19,89 +18,118 @@ global with sharing class GetChildCollection {
             SObject inputRecord = curRequest.inputRecord;
             String inputRecordId = curRequest.inputRecordId;
 
-            SObjectType recordType;
-            SObjectField lookupField;
-            
+            SObjectType parentSobjType;
```

Comment:
Renamed this variable. "recordType" has a specific meaning in Salesforce-speak that this variable doesn't conform with.